list_of_subprojects.md

```diff
@@ -0,0 +1,71 @@
+
+- JupyterLab (https://github.com/jupyterlab/jupyterlab)
+- JupyterHub and Binder 
+  - https://github.com/jupyterhub/jupyterhub
+  - https://github.com/jupyterhub/binder
+  - https://github.com/jupyterhub/binderhub
+- Voilà (https://github.com/voila-dashboards/voila)
+- Jupyter Server (https://github.com/jupyter-server/jupyter_server)
+- Jupyter Widgets (https://github.com/jupyter-widgets)
+- Jupyter Notebook (https://github.com/jupyter/notebook)
+- Jupyter Kernels
+  - jupyter-xeus (https://github.com/jupyter-xeus/)
+  - IPykernel (https://github.com/ipython/ipykernel)
+  - IPython (https://github.com/ipython/ipython)
+- Jupyter Foundations
+- Jupyter Standards
+- Jupyter UX, Design and Branding
+- Jupyter Enterprise
+
+### Official Subprojects without SSC representation
+
+The following official Jupyter Subprojects are smaller and less active. As such their formal decision making body will be the SSC and they will not have an independent SSC delegate. Our expectation is that these smaller teams will basically continue working as they do today, making decisions using the consensus seeking phase of the Jupyter Decision Making Guidelines. In the rare situation that a vote is called, the SSC will serve as the voting body and will consult closely with the team. If one of these Subprojects grows or becomes more active, the SSC can, at any time, elect a standalone decision making body for the Subproject, at which point, the Subproject will begin to have an SSC delegate of their own. In all other respects, these Subprojects are full and official Subprojects.
+
+- nbdime (https://github.com/jupyter/nbdime)
+- nbgrader (https://github.com/jupyter/nbgrader)
+- ipyparallel (https://github.com/ipython/ipyparallel)
+- QtConsole (https://github.com/jupyter/qtconsole). Note that QtConsole today is most actively maintained by the Spyder team; as part of this reorganization we will discuss with them whether it's more appropriate to move the project to be fully under their organization.
+- All other repos not listed above and that don't clearly fall into one of the official Subproject GitHub organizations above.
+
+## Notes about Official Jupyter Subprojects
+
+The Official Jupyter Subprojects document proposes a number of changes to how our repositories are organized into official Subprojects and GitHub organizations. This document describes the changes that are being proposed.
+
+- Services
+  - In general Jupyter services such as Binder, nbviewer, and the Jupyter website involve legal, financial, and operational matters. As such, we are proposing that the actual services are managed by working groups that report to the Board of Directors, who can provide the needed support and oversight. For example, if we want to hire full or part time staff to maintain or operate these services, the Board of Directors would be responsible for raising funds, hiring, and managing those staff.
```

Comment:
How do we ensure that sub projects are resourced to meet these new requirements? For example Binder is already running on a shoe string budget (aka, it has no budget at all and only gets cloud credits). Will the board of directors commit to finding resources that support these extra reporting duties described here? How do we ensure that those duties are carried out equitably and fairly?

Reply:
The current plan is precisely for those to be duties of the BoD - it will need to work with the teams to help them raise funds, allocate resources and provide any necessary coordination. Equity and fairness are high-level principles that we hold dear, but I'm not sure that at this stage, where we're outlining big-picture roles and responsibilities, we are equipped to detail _how_ those principles should be implemented. I think that will be the job of the BoD once it comes online, and we'll need to be vigilant that indeed those ideals are being met in practice. But I don't quite see how to draft a short, high-level organizational PR right now that --aside from stating those principles, which is important-- covers their implementation in detail.